src/storage-types/src/sources/postgres.rs

```diff
@@ -157,18 +157,58 @@ impl<C: ConnectionAccess> crate::AlterCompatible for PostgresSourceConnection<C>
     }
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Arbitrary)]
+pub enum CastType {
+    /// This cast is corresponds to its type in the upstream system.
```

Comment:
```suggestion
    /// This cast corresponds to its type in the upstream system.
```

Reply:
Ugh--good find. Not going to go back through CI to fix this but will sweep it up in some other changes.